spring-cloud-azure-config/pom.xml

```diff
@@ -45,5 +57,9 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
```

Comment:
This should already include `spring-boot-autoconfigure`

Reply:
As it's `optional` in `spring-boot-autoconfigure`, so I think we have to explicitly include it here, though in this repo's case, these jackson library has also been transitively dependent by other azure modules.

Reply:
Removed extra dependency.